checkout/other-features.md

```diff
@@ -51,7 +51,7 @@ underlying payments or transaction.
 | :------: | :------------------ | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | ✔ ︎︎︎︎︎  | `hostUrls`          | `array`  | The array of URIs valid for embedding of Swedbank Pay Hosted Views.                                                                                                                                                                                                                                      |
 |  ✔︎︎︎︎︎  | `completeUrl`       | `string` | The URL that Swedbank Pay will redirect back to when the payer has completed his or her interactions with the payment. This does not indicate a successful payment, only that it has reached a final (complete) state. A `GET` request needs to be performed on the payment order to inspect it further. |
-|  ✔︎︎︎︎︎  | `termsOfServiceUrl` | `string` | The URI to the terms of service document the payer must accept in order to complete the payment. **HTTPS is a requirement**.                                                                                                                                                                             |
+|  ✔︎︎︎︎︎  | `termsOfServiceUrl` | `string` | The URI to the terms of service document the payer must accept in order to complete the payment. **HTTPS is a requirement**. Note that this is not required unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted in the request. This is the Merchants, not the Swedbank Pay Terms of Service.                                                                                                                                                                             |
```

Comment:
This description does not match the one above. Do they need to be different, or should we perhaps add a `field-description-termsofserviceurl.md` include for `termsOfServiceUrl` so they become equal?